src/plots/cartesian/axes.js

```diff
@@ -755,6 +755,9 @@ axes.autoTicks = function(ax, roughDTick) {
     // prevent infinite loops
     if(ax.dtick === 0) ax.dtick = 1;
 
+    // prevent issue https://github.com/plotly/plotly.js/issues/3224
+    if(Number.isNaN(ax.dtick)) ax.dtick = 1;
```

Comment:
Do we know under which circumstance `ax.dtick` comes through as `NaN` here?

Reply:
> It could simply be triggered for example using gl3d_ibm-plot by panning the graph to outside.

Hmm. I can't replicate. Can you take a screenshot of 

```js
gd._fullLayout.scene.camera.eye
```

in the console once after that error appears.

-----

> I am wondering now that we may need to replace the condition checks in few other places there too?

I'm actually thinking the opposite: we shouldn't be calling `Axes.autoTicks` in the first place whenever `ditck===NaN`. But first, I'll like to know how this happens.

Reply:
> I'm actually thinking the opposite: we shouldn't be calling `Axes.autoTicks` in the first place whenever `ditck===NaN`. But first, I'll like to know how this happens.

Agreed. For one thing, `dtick=1` would draw [1000 ticks](https://github.com/plotly/plotly.js/blob/2009f0988fad6e0f281b941178b82e5162281c9b/src/plots/cartesian/axes.js#L561) on date or otherwise large-range axes which would bog things down and look ugly. What's `roughDTick` when we get `NaN` here, and why?

Reply:
@etpinard here is a [demo](https://codepen.io/MojtabaSamimi/pen/bQpvzJ)

Reply:
And this is [another example](https://codepen.io/MojtabaSamimi/pen/KrzRVd) example now without date and logs.

Reply:
Right, this is consistent with what I was thinking: we get those `Uncaught ax.dtick error: NaN` errors in the console in situations where ticks can't be seen.